src/fireflyClient.ts

```diff
@@ -920,6 +922,38 @@ export class FireflyClient {
     return response;
   };
 
+  /**
+   * Gets Open Orders placed by the user. Returns the first 50 orders by default.
+   * @param params of type OpenOrderRequest,
+   * @returns OrderResponse array
+   */
+  getUserOpenOrders = async (params: GetOpenOrderRequest) => {
+    const response = await this.apiService.get<GetOrderResponse[]>(
+      SERVICE_URLS.USER.OPEN_ORDERS,
+      {
+        ...params,
+      },
+      { isAuthenticationRequired: true }
+    );
+    return response;
+  };
+
+   /**
+   * Gets Orders by type and statuses. Returns the first 50 orders by default.
+   * @param params of type OrderByTypeRequest,
+   * @returns OrderResponse array
+   */
+   getUserOrdersByType = async (params: GetOrderByTypeRequest) => {
```

Comment:
There should only one route, either openOrders or orders by orderType as both have the same implementation on dapi as mentioned [here](https://www.notion.so/seedlabs/API-Clean-Up-0fe8c9749da748738a6d630f42ebac8e)

Reply:
I have removed the getOpenOrders route in favor of order-by-type route; also the order-by-type route originally missing parentAddress input param in dAPI for which [PR](https://github.com/fireflyprotocol/dapi-server/pull/1092)  is already raised.